.travis.yml

```diff
@@ -5,16 +5,38 @@ rvm:
   2.6.1
 addons:
   chrome: stable
+  apt:
+    packages:
+    - libsndfile-dev
+    - lame
+    # For VIPS
+    - libexpat1-dev
+    - gettext
+    - libglib2.0-dev
+    - liborc-0.4-dev
+    - libfftw3-dev
+    - liblcms2-dev
+    - libmagickwand-dev
+    - libopenexr-dev
+    - libcfitsio3-dev
+    - libgif-dev
+    - libgsf-1-dev
+    - libmatio-dev
+    - libopenslide-dev
+    - libpango1.0-dev
+    - libpoppler-glib-dev
+    - librsvg2-dev
+    - libwebp-dev
 services:
-  - mysql
+- mysql
 cache:
-- bundler
-- yarn
-- npm
+  bundler: true
+  yarn: true
+  directories:
+  - $HOME/vips
 before_install:
-- npm i -g node@latest
```

Comment:
This used to be required, but seems like everything is happy now?

Reply:
Yeah, I think Ubuntu used to ship with an ancient version of Node. But probably good enough for now.